### PR TITLE
[WIP] feature: add an option to not require checksum

### PR DESCRIPTION
**Pull request by sarfata**
By default sentences without a checksum are considered to have an
invalid checksum (like before). With the new option `requireChecksum`
set to false, they will be accepted but we will still validate checksums
on lines that have one.

Tests for this are in signalk-parser-nmea0183 (this is where existing
tests were already placed).

### CODE REVIEW

**Comment by sarfata** [comment]
Wait before merging. Need to handle sentence that are not terminated by a '*' too.

**Comment by tkurki** [comment]
Please remove the [WIP] in PR title when done and ping.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@signalk/nmea0183-utilities",
-  "version": "0.6.0",
+  "version": "0.7.0",
   "description": "Various utilities for transforming NMEA0183 units into SI units for use in SK.",
```
Comment (tkurki) [inline package.json:4]: I’d like to keep version bumps separate commits. You don’t know when the commit will be part of a published package. If you want to indicate how a commit or pr should be treated per semantic versioning one way to do that would be https://github.com/conventional-changelog/standard-version : write strictly formatted messages.

**Comment by tkurki** [comment]
Closing due to inactivity.
